test/api/extensions/customer-ext.spec.js

```diff
@@ -1,56 +1,154 @@
-const supertest = require("supertest")
-const app = require("../../../src/server.js")
-const { createCustomer } = require("../../helpers/createCustomer.js")
-
-describe("Customer Endpoint", () => {
-    describe("PUT /customers/:id", () => {
-        it("can update a customers contact info when a contact property exists on the request body", async () => {
-            const customer = await createCustomer("John", "123456", "john@test.com")
-
-            const request = {
-                name: "Jane",
-                contact: {
-                    phone: "789",
-                    email: "jane@test.com"
-                }
-            }
-
-            const response = await supertest(app)
-                .put(`/customers/${customer.id}`)
-                .send(request)
-
-            expect(response.status).toEqual(201)
-            expect(response.body.customer).not.toEqual(undefined)
-            expect(response.body.customer.name).toEqual(request.name)
-            expect(response.body.customer.contact).not.toEqual(undefined)
-            expect(response.body.customer.contact.phone).toEqual("789")
-            expect(response.body.customer.contact.email).toEqual("jane@test.com")
-        })
-
-        it('will return 404 if the customer is not found', async () => {
-            const request = {
-                name: "Jane",
-            }
-
-            const response = await supertest(app)
-                .put(`/customers/10000`)
-                .send(request)
-
-            expect(response.status).toEqual(404)
-            expect(response.body).toHaveProperty('error')
-        })
-
-        it("will return 400 when there are missing fields in the request body", async () => {
-            const customer = await createCustomer("John", "123456", "john@test.com")
-
-            const request = {}
-
-            const response = await supertest(app)
-                .put(`/customers/${customer.id}`)
-                .send(request)
-
-            expect(response.status).toEqual(400)
-            expect(response.body).toHaveProperty('error')
-        })
-    })
-})
+const supertest = require("supertest");
+const app = require("../../../src/server.js");
+const { createCustomer } = require("../../helpers/createCustomer.js");
+const { createScreen } = require("../../helpers/createScreen.js");
+const { createMovie } = require("../../helpers/createMovie.js");
+
+// describe("Customer Endpoint", () => {
```

Comment:
Did this test stop passing after working on the exercise?

Reply:
I never got to work on the customer endpoint in the extensions. I have removed this test file in the final commit.